client/packages/common/src/intl/locales/en/common.json

```diff
@@ -315,15 +315,20 @@
   "error.not-editable": "Stocktake is not editable.",
   "error.not-most-recent-given-dose": "This is not the most recent vaccination. Please edit the most recent vaccination if a dose has not been given.",
   "error.order-not-found": "Internal Order not found",
+  "error.ordering-too-many-items": "You cannot finalise an emergency order with more than {{ maxItems }} items. Reduce the number of items and try again.",
```

Comment:
I think just the first sentence "You cannot finalise an emergency order with more than {{ maxItems }} items." would better match the existing error messaging we have.

Reply:
Think it's fine to leave the actionable in there as well? This is what is displayed in OG, just copying it over (:

Reply:
Also wonder if we might need the _one, _few, _many, _other pattern for this translation to cover the case of only 1 item allowed on emergency order (+ few vs many cases for Russian translations)

Reply:
Good thinking (: